src/sass/_shame-wizard.scss

```diff
@@ -0,0 +1,225 @@
+// This file is a place where you put all the code you're not proud of,
+// with the intention of A) isolating it and B) fixing it later. The goal
+// is for this file to be empty, but some times you just have to throw that
+// hack in there to get it working.
+
+// message while loading react app
+.loading-message {
+  text-align: center;
+  padding: 2em 0;
+}
+
+body .row {
+  max-width: 62.5em;
+}
+
+.form-panel {
+  @include media($medium-screen) {
+    max-width: $usa-form-width;
+  }
+}
+
+.main-form,
+.progress-box {
+  button {
+    width: 100%;
+  }
+}
+
+.button-icon{
+  font-size: 1.7rem;
+}
+
+.progress-box {
+  padding: 1rem 2rem;
+  margin: 1.5rem 0;
+  border: 1px solid $color-gray-lightest;
+}
+
+.form-section {
+  display: none;
+}
+
+#content .section {
+  fieldset {
+    h4 {
+      padding: 0;
+    }
+  }
+}
+
+.edit-checkbox {
+  label {
+    background-color: transparent;
+    display: inline-block;
+
+    &:before {
+      display: none !important;
+    }
+
+    &:hover {
+      cursor: pointer;
+      outline: 0;
+    }
+  }
+}
+
+.input-section {
+  margin-bottom: 2em;
+
+  .row {
+    margin: 0;
+  }
+}
+
+.usa-input-error {
+  select {
+    border: 3px solid $color-secondary-dark;
+    width: calc(100% + 1.5rem);
+  }
+}
+
+.va-alert-text {
+  margin: 0;
+}
+
+.hidden {
+  display: none !important;
+}
+
+/* inline declarations moved here as a result of CSP rules */
+
+.csp-inline-patch-L2-section-landing {
+  margin-bottom: 1em;
+}
+
+.csp-inline-patch-L2-section-landing-1 {
+  display: inline-block;
+  margin-bottom: 0em;
+}
+
+.csp-inline-patch-L2-section-landing-2 {
+  display: inline-block;
+}
+
+.csp-inline-patch-footer {
+  display: none;
+  visibility: hidden;
+}
+
+.csp-inline-patch-header {
+  margin: 0;
+  padding: 0;
+  display: inline;
+}
+
+.csp-inline-patch-modal {
+  display: none;
+}
+
+.csp-inline-patch-page-breadcrumbs {
+  background-color: $color-warning-message;
+  text-align: center;
+}
+
+.csp-inline-patch-page-breadcrumbs-2 {
+  margin: 0;
+}
+
+.csp-inline-patch-page-react {
+  padding: 2em 0;
+}
+
+.csp-inline-patch-404 {
+  margin: 0;
+  padding: 0;
+  display: inline;
+}
+
+.csp-inline-patch-application {
+  padding: 2em 0;
+}
+
+.csp-inline-patch-logout {
+  padding: 2em 0;
+}
+
+// /education/apply/ uses this expander button for navigation
+// Where else can I put this? Also, it could use a better name.
+.expander-button {
```

Comment:
The original intention was for this to be a copy to swap in or out, but your idea to create a module is a better way to go about this.